user-sample/src/sample/CreateTransferTokenSample.cs

```diff
@@ -0,0 +1,158 @@
+            UserMember payer,
+            Alias payeeAlias)
+        {
+            // Set SEPA destination.
+            TransferDestination sepaDestination = new TransferDestination
+            {
+                Sepa = new TransferDestination.Types.Sepa
+                {
+                    Bic = "XUIWC2489",
+                    Iban = "DE89 3704 0044 0532 0130 00"
+                }
+            };
+            // Set the destination and other details.
+            TransferTokenBuilder builder =
+                payer.CreateTransferTokenBuilder(
+                        100.0, // amount
+                        "EUR") // currency
+                    .SetAccountId(payer.GetAccountsBlocking()[0].Id())
+                    .SetToAlias(payeeAlias)
+                    .AddDestination(sepaDestination);
+            // Get the token redemption policy and resolve the token payload.
+            PrepareTokenResult result = payer.PrepareTransferTokenBlocking(builder);
+            // Create the token, signing with the payer's STANDARD-level key
+            Token transferToken = payer.CreateTokenBlocking(result.TokenPayload, Level.Standard);
+            return transferToken;
+        }
+
+        /// <summary>
+        /// Creates a transfer token with a later execution date.
+        /// </summary>
+        /// <param name="payer">payer Token member</param>
+        /// <param name="payeeAlias">payee Token member alias</param>
+        /// <returns>a transfer Token</returns>
+        public static Token CreateTransferTokenScheduled(
+            UserMember payer,
+            Alias payeeAlias)
+        {
+            // We'll use this as a reference ID. Normally, a payer who
+            // explicitly sets a reference ID would use an ID from a db.
+            // E.g., a bill-paying service might use ID of a "purchase".
+            // We don't have a db, so we fake it with a random string:
+            string purchaseId = Util.Nonce();
+            // Set the details of the token.
+            TransferTokenBuilder builder = payer.CreateTransferTokenBuilder(
+                    10.0, // amount
+                    "EUR") // currency
+                           // source account:
+                .SetAccountId(payer.GetAccountsBlocking()[0].Id())
+                // payee token alias:
+                .SetToAlias(payeeAlias)
+                // optional description:
+                .SetDescription("Book purchase")
+                // ref id (if not set, will get random ID)
+                .SetRefId(purchaseId)
+                // set the transfer to execute in 30 days
+                .SetExecutionDate(DateTime.UtcNow.AddDays(30).ToString("yyyy-MM-dd"));
+            // Get the token redemption policy and resolve the token payload.
+            PrepareTokenResult result = payer.PrepareTransferTokenBlocking(builder);
+            // Create the token: Default behavior is to provide the member's signature
```

Comment:
How about:
"The default redemption policy is to provide ..."
You mentioned "Get the redemption policy" above but nowhere to see the redemption policy in use from the comment.

Reply:
Used the same format as used in SDK java